docs/definition.md

```diff
@@ -180,6 +180,573 @@ There are a few terms you should be familiar with:
 7. `Success`, `Failure` and `Result` are markers returned by step
    methods to change business process execution path.
 
+### Invalid story definitions
```

Comment:
Does it make more sense to move it to the FAQ section?

Reply:
No, this should be present here so our users will know what not to do.
This is as important as what you can do.